docs/build/linux-ubuntu.md

```diff
@@ -127,20 +135,32 @@ Many dependencies are only required for specific projects.
 
 ### Base Dependencies
 
-**Environment variables**:
+**Environment variables:**
 
+* --
 
 ## Notes on Environment Variables
 
 ### JAVA_HOME
 
-This environment variable tells Gradle what version of Java to use for building Keyman for Android.
-OpenJDK 11 is used for master. It's recommended to set the environment variables to:
+This environment variable tells Gradle what version of Java to use for building
+Keyman for Android. OpenJDK 11 is used for master.
+
+It's recommended to set the environment variables to:
+
 ```bash
-EXPORT JAVA_HOME="[path to OpenJDK 11]"
+export JAVA_HOME="[path to OpenJDK 11]"
 ```
 
 Also edit `/etc/profile.d/jvm.sh` as sudo:
+
 ```bash
-EXPORT JAVA_HOME="[path to OpenJDK 11]"
+export JAVA_HOME="[path to OpenJDK 11]"
 ```
+
+**Multiple versions of Java:** If you need to build Keyman for Android 16.0 or
+older versions, you can set `JAVA_HOME_11` to the OpenJDK 11 path and
+`JAVA_HOME` to the OpenJDK 8 path. This will build both versions correctly
+from command line. But note that you do need to update your `JAVA_HOME` env
+var to the associated version before opening Android Studio and loading any
+Android projects. `JAVA_HOME_11` is mostly used by CI.
```

Comment:
I don't remember which of my PR's it was, but I think @mcdurdin said we won't bother documenting Java requirementes for 16.0 (since this is master branch)

Reply:
I think the behavior of `JAVA_HOME_11` is still worth documenting and Keyman 16 serves as example.